screens/ChatScreen.tsx

```diff
@@ -44,44 +52,166 @@ export default function ChatScreen({
   }, []);
 
   const [count, setCount] = useState(0);
+  const [messages, setMessages] = React.useState<IMessage[]>([
+    {
+      _id: 1,
+      text: "Hello developer",
+      createdAt: new Date(),
+      user: {
+        _id: 4,
+        name: "GiftedChat",
+        avatar: "https://placeimg.com/140/140/any",
+      },
+    },
+    {
+      _id: 2,
+      text: "Hello developer",
+      createdAt: new Date(),
+      user: {
+        _id: 5,
+        name: "GiftedChat",
+        avatar: "https://placeimg.com/140/140/any",
+      },
+    },
+    {
+      _id: 3,
+      text: "Hello developerewew/nwqqwqmscmlkdcccccccccccccccmxxxxxxxxxxxxxxxcsdcs",
+      createdAt: new Date(),
+      user: {
+        _id: 6,
+        name: "GiftedChat",
+        avatar: "https://placeimg.com/140/140/any",
+      },
+    },
+  ]);
+  const onSend = (newMessages: IMessage[] = []) =>
+    setMessages(GiftedChat.append(messages, newMessages));
+  function renderMessage(props) {
+    return (
+      <Message
+        {...props}
+        containerStyle={{
+          left: { marginVertical: 10 },
+        }}
+      />
+    );
+  }
+  function renderBubble(props) {
+    return (
+      <Bubble
+        {...props}
+        wrapperStyle={{
+          left: {
+            backgroundColor: "#ECECEC",
+            borderRadius: 12,
+            padding: 6,
+          },
+          right: {
+            backgroundColor: "#ECECEC",
+            marginVertical: 10,
+            borderRadius: 12,
+            borderTopRightRadius: 12,
+            padding: 6,
+          },
+        }}
+        textStyle={{
+          left: {
+            color: "#000000",
+            fontSize: 18,
+          },
+          right: {
+            color: "#000000",
+            fontSize: 18,
+          },
+        }}
+        timeTextStyle={{
+          left: {
+            color: "gray",
+          },
+          right: {
+            color: "gray",
+          },
+        }}
+      />
+    );
+  }
+  function renderInputToolbar(props) {
+    return (
+      <SafeAreaView style={styles.filter}>
+        <ButtonBanner count={count} setCount={setCount} data={FILTER} />
+        <SafeAreaView style={{ flexDirection: "row" }}>
+          <TextInput
+            style={[styles.input, { marginBottom: keyboardOffset - 125 }]}
+            onChangeText={(text) => setText(text)}
+            value={text}
+            //multiline={true}
+          />
+          <TouchableOpacity
+            style={{ marginRight: 20, marginLeft: "auto", marginTop: 2.5 }}
+            onPress={() => {
+              if (text.length > 0) {
```

Comment:
it'd be better if you also detect if string is all spaces: https://stackoverflow.com/questions/10261986/how-to-detect-string-which-contains-only-spaces